packages/particle-cli/app/storybook/main.js

```diff
@@ -0,0 +1,36 @@
+const path = require('path')frontendFramework
+
+const APP_COMPONENT_LIBRARY = path.resolve(__dirname, '../../src/default')
+const particle = require('../../particle')
+
+const dev = {}
+const prod = {}
+
+const cssMode = process.env.NODE_ENV === 'production' ? 'extract' : 'hot'
+
+module.exports = {
+  addons: ['@storybook/addon-viewport","@storybook/addon-knobs","@storybook/addon-actions","@storybook/addon-a11y","@storybook/addon-links"],
+  stories: ['./stories/**/*.story.tsx"],
+  webpackFinal: (config) => {
+    /**
+     * Delete the CSS management rules from Storybook.
+     * Also delete the file-loader ruleset from Storybook in favor of Particle
+     * Particle.js owns that process.
+     */
+    // eslint-disable-next-line no-param-reassign
+    config.module.rules.splice(2, 2)
+    /**
+     * Delete the ProgressPlugin from Storybook for CI to remove
+     * log file spam.
+     */
+    if (process.env.CI === 'true') {
+      // eslint-disable-next-line no-param-reassign
+      config.plugins.splice(4, 1)
+    }
+    return particle(
```

Comment:
Have we re-implemented `particle` in 11? May be premature since we had not yet settled on a Webpack strategy in the context of running these tools from the root of an Octane (or Gatsby) project.

Reply:
This was from sentinel API. Not sure if this is the approach we want. Going to look at things today. I'll add a todo into the branch code here to come back and resolve the pathing.

Since we have not finalized base config, I brought this in for now.